### DISCUSSION TITLE
How to set up a user to register or login in to an account before they can access the page?

### DISCUSSION THREAD

**Discussion by Kelly-ZH** [requarks/wiki#6930] [Help / Questions] [answered]
According to the [introduction](https://docs.requarks.io/install/ubuntu),  I have installed wiki app, and I can visit the page, but user doesn't  register or login an account, he or she also can browse the page, that's what I don't want. 

I looked at the version iterations of wiki.js, and I rollbacked the code as[ this](https://github.com/requarks/wiki/commit/fe890979afe04e43d90486aaaeda4abd587d62b9) , but it doesn't work.

**So,  my question is how to set up a user to register or login in to an account before they can access the page?**
### PoC

**Comment by fnafox** [comment, marked as answer]
Remove read permissions from the guest group to disable anonymous access. After that set up user/group permissions according to your needs.

**Comment by Kelly-ZH** [comment]
Thank you for your reply. I have solved the problem. Now I have a new question, as shown in the figure below, it supports lots of types to push the target to backup data, but only a little support pulling from the target to restore the data. My new question is how to backup and restore wiki pages? 
![1705020388(1)](https://github.com/requarks/wiki/assets/56919535/34ddb01f-73c1-4fe1-baf3-9a28b4f18f51)

**Reply by fnafox** [reply]
if you are interested in specifically restoring wiki contents from an archive, unpack the archive into the target specified in the local file system storage option then use the `Import Everything` button.

if your goal is to backup the entire wiki including configuration, then i'd suggest making regular database dumps as everything is contained there. the documentation page for transferring the wiki install explains how to restore it https://docs.requarks.io/en/install/transfer

**Reply by fnafox** [reply]
your logo must be guest readable for it to be visible before login, you just need to explicitly adjust permissions for the logo

**Reply by Kelly-ZH** [reply]
ok, i think it's a good solution.  Then I use a single Docker deployment like [this](https://docs.requarks.io/install/ubuntu), and the peak page access may reach thousands of levels. Can I handle such traffic?

**Reply by fnafox** [reply]
that's not something i can answer easily, i know deepin linux uses wikijs as a public wiki so i imagine it should be fine
however, i suggest doing performance testing before deploying it at production scale

**Reply by Kelly-ZH** [reply]
ok, i will test before deploying it at production scale.  Anyway, thank you for your answer.

**Reply by Kelly-ZH** [reply]
Hi，how to change these small logos at browser tab page?
![image](https://github.com/requarks/wiki/assets/56919535/22a418ba-2cc8-45dd-921f-0d5b743df422)

**Reply by fnafox** [reply]
Unfortunately there is no easy way to do this, you have to replace all the relevant assets in the wiki directory with your images.
If you're using docker just externally mounting the relevant files is probably the best approach.
```
assets/favicon.ico
assets/favicons/
```
Remember to back them up before any wiki updates as they will get overriden.

**Reply by Kelly-ZH** [reply]
I see, i wil try it. Thank you.
